client/src/components/SignUpPhonePreferences.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+export class SignUpPhonePreferences extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <form className="ui form">
+        <h4 className="ui dividing header">Call Preferences</h4>
+        <div className="field">
+          <label>Scheduled Call Time</label>
+          <div className=" three fields">
+            <div className="two wide field">
+              <select className="ui fluid search dropdown">
+                <option value="">Hour</option>
+                <option value="1">1</option>
```

Comment:
I would suggest using map to render time options - will reduce the amount of code needed here